Event-Framework-Tests.postman_collection.json

```diff
@@ -246,6 +246,859 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Transfer Prepare Error - Invalid FSP",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "9d58aca2-4bdc-4c4b-b12b-b70c76f3c36d",
+						"exec": [
+							"var uuid = require('uuid');",
+							"var generatedUUID = uuid.v4();",
+							"",
+							"pm.environment.set('transfer_ID', generatedUUID);",
```

Comment:
Should these not be set in the env file?

Reply:
uuid, generatedUUID and transfer_ID - always used this way in the gold path postman collection. Efk URI can be moved to env file.